firebase-perf/src/main/java/com/google/firebase/perf/metrics/validator/PerfMetricValidator.java

```diff
@@ -143,9 +145,9 @@ public static String validateMetricName(@Nullable String str) {
   public static String validateAttribute(@NonNull Map.Entry<String, String> attribute) {
     String key = attribute.getKey();
     String value = attribute.getValue();
-    if (key == null) {
+    if (key == null || key.length() == 0) {
       return "Attribute key must not be null";
```

Comment:
nit: return "Attribute key must not be null or an empty string.

Reply:
Updated! Done.